spec/unit/server_create_command_spec.rb

```diff
@@ -110,89 +108,13 @@ class ServerCreate < Chef::Knife::Cloud::ServerCreateCommand
     include Chef::Knife::Cloud::ServerCreateOptions
   end
 
-  describe "#bootstrap options" do
-
-    it "set chef config knife options" do
+  describe "Bootstrap Protocol option" do
+    it "not to be set in chef config knife options" do
```

Comment:
can the describe and it line descriptions be improved here, I'm not really sure what this test is supposed to be doing.

Reply:
It is just to check by any chance we are not sending `--bootstrap-protocol` option to @bootstarp object.